### PR TITLE
add bond-selector info to SVGs

### PR DESCRIPTION
**Pull request by greglandrum**
Modifies `tagAtoms()` to add bond selector class info as well as atom selector info.

Also includes a bit more documentation

### CODE REVIEW

File: Code/GraphMol/MolDraw2D/MolDraw2DSVG.cpp [hunk 1]
```diff
@@ -457,6 +457,28 @@ void MolDraw2DSVG::addMoleculeMetadata(const std::vector<ROMol *> &mols,
...
+    auto a2pos = getDrawCoords(atomCoords()[bond->getEndAtomIdx()]);
+    d_os << "<path "
+         << " d='M " << a1pos.x << "," << a1pos.y << " L "<<a2pos.x<<","<<a2pos.y<<
+         "'";
+    d_os << " class='bond-selector bond-" << this_idx;
+    if (d_activeClass != "") {
+      d_os << " " << d_activeClass;
+    }
+    d_os << "'";
+    d_os << " style='fill:#fff;stroke:#fff;stroke-width:5px;fill-opacity:0;"
```
Comment (bp-kelley) [inline Code/GraphMol/MolDraw2D/MolDraw2DSVG.cpp]: Perhaps we should use a stroke width of max(lineWidth(), 5).

File: Code/GraphMol/MolDraw2D/MolDraw2DSVG.cpp [hunk 2]
```diff
@@ -457,6 +457,28 @@ void MolDraw2DSVG::addMoleculeMetadata(const std::vector<ROMol *> &mols,
...
+            "stroke-opacity:0' ";
+    // for (const auto &event : events) {
```
Comment (bp-kelley) [inline Code/GraphMol/MolDraw2D/MolDraw2DSVG.cpp]: Any reason to leave the commented out events?

File: Code/GraphMol/MolDraw2D/MolDraw2DSVG.cpp (hunk 1 above)
Comment (greglandrum) [reply Code/GraphMol/MolDraw2D/MolDraw2DSVG.cpp]: Probably better to do lineWidth()+2 
I will try that out on Monday and update

Comment (greglandrum) [reply Code/GraphMol/MolDraw2D/MolDraw2DSVG.cpp]: Probably better to do lineWidth()+2 
I will try that out on Monday and update

File: Code/GraphMol/MolDraw2D/MolDraw2DSVG.cpp (hunk 2 above)
Comment (greglandrum) [reply Code/GraphMol/MolDraw2D/MolDraw2DSVG.cpp]: Just lazy...

**Pull request merged**